modules/Bio/EnsEMBL/Analysis/Hive/RunnableDB/HiveCreatePlaceholderSQL.pm

```diff
@@ -0,0 +1,99 @@
+
+=head1 CONTACT
+
+Please email comments or questions to the public Ensembl
+developers list at <http://lists.ensembl.org/mailman/listinfo/dev>.
+
+Questions may also be sent to the Ensembl help desk at
+<http://www.ensembl.org/Help/Contact>.
+
+=cut
+
+package Bio::EnsEMBL::Analysis::Hive::RunnableDB::HiveCreatePlaceholderSQL;
+
+use warnings;
+use strict;
+
+use parent ('Bio::EnsEMBL::Analysis::Hive::RunnableDB::HiveBaseRunnableDB');
+
+=head2 fetch_input
+
+    Description : Implements fetch_input() interface method of Bio::EnsEMBL::Hive::Process that is used to read in parameters and load data.
+                  Here we have nothing to do.
+
+=cut
+
+sub fetch_input {
+}
+
+=head2 run
+
+ Arg [1]    : None
+ Description: Creates an sql command add placeholder location information to the meta table
+ Returntype : None
+ Exceptions : None
+
+=cut
+
+sub run {
+  my ($self) = @_;
+
+  my $core_dba = $self->hrdb_get_dba($self->param('input_db'));
+  my $sa = $core_dba->get_SliceAdaptor;
+
+  my $sth_longest = $core_dba->dbc->prepare('select seq_region_id from seq_region order by length desc limit 10');
+  $sth_longest->execute;
+
+  my $sample_transcript;
+ LOOP: while (my $seq_region_id = $sth_longest->fetchrow_array) {
+    my $region = $sa->fetch_by_seq_region_id($seq_region_id);
+    my @transcripts = @{$region->get_all_Transcripts_by_type('protein_coding')};
+
+  TRANSCRIPT:foreach my $transcript (@transcripts){
+      my $supporting_features = $transcript->get_all_supporting_features;
+      foreach my $support (@$supporting_features){
+        if ($support->hcoverage() >= 99 && $support->percent_id() >= 75){
+          $sample_transcript=$transcript;
+          last LOOP;
+        }
+        else{
+          next TRANSCRIPT;
+        }
+      }
+    }
+  }#end while
+
+  if ($sample_transcript){
+    my $db_name = $core_dba->dbc->dbname;
+    my $sample_gene = $sample_transcript->get_Gene;
+    my $sample_coord = $sample_gene->seq_region_name().':'.$sample_gene->seq_region_start().'-'.$sample_gene->seq_region_end();
+
+    my @sql = ("INSERT INTO meta (species_id, meta_key, meta_value) VALUES (1, 'sample.location_param', '".$sample_coord."')","INSERT INTO meta (species_id, meta_key, meta_value) VALUES (1, 'sample.location_text', '".$sample_coord."')","INSERT INTO meta (species_id, meta_key, meta_value) VALUES (1, 'sample.gene_param', '".$sample_gene->stable_id()."')","INSERT INTO meta (species_id, meta_key, meta_value) VALUES (1, 'sample.gene_text', 'ensembl_gene')","INSERT INTO meta (species_id, meta_key, meta_value) VALUES (1, 'sample.transcript_param', '".$sample_transcript->stable_id()."')","INSERT INTO meta (species_id, meta_key, meta_value) VALUES (1, 'sample.transcript_text', 'ensembl_transcript')","INSERT INTO meta (species_id, meta_key, meta_value) VALUES (1, 'sample.search_text', 'ensembl_gene')");
+
+    my $output_hash = {};
+    $output_hash->{'sql_command'} = \@sql;
+    $self->dataflow_output_id($output_hash,1);
```

Comment:
Unless there is really no other way, we should not use branch 1. _branch_to_flow_to can be used and if really needed, set to 1 in param_defaults.
Same remark about dataflow_output_id as the other module